src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/CosmosParameterStore.cs

```diff
@@ -0,0 +1,29 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Health.Fhir.Core.Features.Parameters;
+
+namespace Microsoft.Health.Fhir.CosmosDb.Features.Storage
+{
+    public class CosmosParameterStore : IParameterStore
```

Comment:
Does this class exist for compatibility or are we planning on using the parameter store for cosmos?

Reply:
Compatibility. I don't think we will use parameters in Cosmos.